sklearn/feature_extraction/hashing.py

```diff
@@ -41,10 +41,12 @@ class FeatureHasher(BaseEstimator, TransformerMixin):
         The number of features (columns) in the output matrices. Small numbers
         of features are likely to cause hash collisions, but large numbers
         will cause larger coefficient dimensions in linear learners.
+
```

Comment:
I'm neither here nor there about the blank lines. But they were one reason I've delayed some automatic docstring fixing utilities...!